src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+    private final Bulldozer bulldozer1 = new Bulldozer();
+    private final Bulldozer bulldozer2 = new Bulldozer();
+    private final Bulldozer bulldozer3 = new Bulldozer();
+
+    private final List<Bulldozer> bulldozers = List.of(bulldozer1, bulldozer2, bulldozer3);
+
+    @Override
+    public List<Bulldozer> get() {
+        return bulldozers;
```

Comment:
now this class will always return the same 3 bulldozers. It's better to make them as local variables and always return new list with new bulldozers. That's default behaviour for producer